### PR TITLE
Street 2 home - first draft

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/grda_warehouse/warehouse_reports/report_definition.rb [hunk 1]
```diff
@@ -841,6 +841,22 @@ def self.report_list # rubocop:disable Metrics/AbcSize
           health: false,
         }
       end
+      if RailsDrivers.loaded.include?(:boston_reports)
+        r_list['Performance'] << {
+          url: 'boston_reports/warehouse_reports/street_to_homes',
+          name: _('Street to Home'),
+          description: 'Boston specific report to track progress for the Street to Home initiative',
```
Comment (awisema) [inline app/models/grda_warehouse/warehouse_reports/report_definition.rb]:
```suggestion
          description: 'Boston-specific report to track progress for the Street to Home initiative',
```

File: app/models/grda_warehouse/warehouse_reports/report_definition.rb [hunk 2]
```diff
@@ -841,6 +841,22 @@ def self.report_list # rubocop:disable Metrics/AbcSize
...
+          limitable: false,
+          health: false,
+        }
+        r_list['Performance'] << {
+          url: 'boston_reports/warehouse_reports/configs',
+          name: _('Boston Reports Configuration'),
+          description: 'Report configuration for Boston specific reports',
```
Comment (awisema) [inline app/models/grda_warehouse/warehouse_reports/report_definition.rb]:
```suggestion
          description: 'Report configuration for Boston-specific reports',
```

File: app/jobs/background_render/street_to_home_job.rb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+class BackgroundRender::StreetToHomeJob < BackgroundRenderJob
+  def render_html(partial:, filters:, user_id:)
+    current_user = User.find(user_id)
+    @filter = ::Filters::FilterBase.new(user_id: user_id).set_from_params(JSON.parse(filters).with_indifferent_access)
+    @comparison_filter = @filter.to_comparison
+    set_report
+    @section = @report.class.available_section_types.detect do |m|
+      m == partial
+    end
+    @section = 'overall' if @section.blank? && params.require(:partial) == 'dashboard'
```
Comment (awisema) [inline app/jobs/background_render/street_to_home_job.rb]: This is 'dashboard' here and 'overall' in `street_to_homes_controller.rb:61` should these be different, or match?

File: drivers/boston_reports/app/controllers/boston_reports/warehouse_reports/street_to_homes_controller.rb [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+    private def report_class
+      BostonReports::StreetToHome
+    end
+
+    def section
+      @section = @report.class.available_section_types.detect do |m|
+        m == params.require(:partial).underscore
+      end
+      @section = 'overall' if @section.blank? && params.require(:partial) == 'overall'
```
Comment (awisema) [inline drivers/boston_reports/app/controllers/boston_reports/warehouse_reports/street_to_homes_controller.rb]: See `street_to_home_job.rb:16`

**Review by awisema — APPROVED** [review]

File: app/jobs/background_render/street_to_home_job.rb (hunk 3 above)
Comment (eanders) [reply app/jobs/background_render/street_to_home_job.rb]: I'll double check.  Thanks.

**Pull request merged**